gitlab/objects.py

```diff
@@ -851,7 +851,8 @@ class ProjectHook(GitlabObject):
     requiredUrlAttrs = ['project_id']
     requiredCreateAttrs = ['url']
     optionalCreateAttrs = ['push_events', 'issues_events',
-                           'merge_requests_events', 'tag_push_events']
+                           'merge_requests_events', 'tag_push_events',
+                           'build_events', 'enable_ssl_verification']
```

Comment:
In gitlab-ce code:

https://gitlab.com/gitlab-org/gitlab-ce/blob/master/lib/api/project_hooks.rb#L48-49